sql/2021/javascript/lighthouse_unminified_js_requests.sql

```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# Percent of requests with and without minification
```

Comment:
This query looks like a copy of the last one, which seems like an error from the 2020 version too: https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/javascript/lighthouse_unminified_js_requests.sql

Reply:
@rviscomi What's the error with this one? I can run this and get the results fine. (running query on table `httparchive.sample_data.lighthouse_mobile_*`)

Reply:
This query appears to be implemented identically to lighthouse_unminified_js_bytes.sql, despite being named lighthouse_unminified_js_requests.sql. Shouldn't it measure requests?